R/flatten.R

```diff
@@ -52,12 +52,24 @@
 #' Since `join = nest_join()` does not make sense in this direction (LHS = child table, RHS = parent table: for valid key constraints
 #' each nested column entry would be a tibble of one row), an error will be thrown if this method is chosen.
 #'
+#' Note that the difference between `dm_flatten_to_tbl()` and
```

Comment:
It might be clearer to not use former/latter but to repeat the names instead.

Reply:
And maybe even to use a list with two items?

Reply:
Good idea. Done!